common/protob/messages-tron.proto

```
@@ -104,8 +105,19 @@
             optional uint64 balance = 3;
             optional string receiver_address = 4;
         }
-
+        message TronCancelAllUnfreezeV2Contract {
+        }
+        // Vote Witness Contract
+        message TronVoteWitnessContract {
+            message Vote {
+                required string vote_address = 1;
```

Buf (1.54.0)

Field named "vote_address" should not be required.

(FIELD_NOT_REQUIRED)